packages/url_launcher/lib/url_launcher.dart

```diff
@@ -38,11 +49,15 @@ const MethodChannel _channel = MethodChannel('plugins.flutter.io/url_launcher');
 /// [statusBarBrightness] Sets the status bar brightness of the application
 /// after opening a link on iOS. Does nothing if no value is passed. This does
 /// not handle reseting the previous status bar style.
-Future<void> launch(
+///
+/// Return a Future contains bool value. If ture, launch url is successful; if false,
```

Comment:
This should explain that false is only returned if a universal link cannot be opened, otherwise it's a little confusing when you look at the "retry" logic.